src/components/Results.jsx

```diff
@@ -1,3 +1,79 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+// import ResultCard from './ResultCard';
+import { useHistory, useParams, Link } from 'react-router-dom';
+import heroes from './heroes.json';
+
 export default function Results() {
-  return <div>RESULTS</div>;
+  const { int, spe, str, pow } = useParams();
+  const [myHeroes, setMyHeroes] = useState([]);
+  const history = useHistory();
+
+  const getStat = () => {
+    let stat;
+    if (int > spe && int > str && int > pow) {
+      stat = 'intelligence';
+    }
+    if (spe > int && spe > str && spe > pow) {
+      stat = 'speed';
+    }
+    if (str > int && str > spe && str > pow) {
+      stat = 'strength';
+    }
+    if (pow > int && pow > str && pow > spe) {
+      stat = 'power';
+    } else {
+      stat = 'intelligence';
+    }
```

Comment:
Bug spotted: what if `int === spe === str === pow` ?
Fix: give a default attribute to `stat` in the beginning